pkg/plugin/vm_restore_item_action.go

```diff
@@ -63,6 +63,10 @@ func (p *VMRestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (
 		return nil, errors.WithStack(err)
 	}
 
+	for i := 0; i < len(vm.Spec.Template.Spec.Domain.Devices.Interfaces); i++ {
```

Comment:
dont think you need this commit..

Reply:
I'm not against having "incomplete" commits such as these if they showcase the development history and specifically wanted to make an exception for these as they are from another contributor. That said, we've had this conversation in the past and I'm also ok with a cleaner approach, so I'll squash them as suggested.